modules/images/dominant-color/load.php

```diff
@@ -34,7 +34,7 @@ function dominant_color_metadata( $metadata, $attachment_id ) {
 add_filter( 'wp_generate_attachment_metadata', 'dominant_color_metadata', 10, 2 );
 
 /**
- * Filter various image attributes to add the dominant color to the image
+ * Filter various image attributes to add the dominant color to the image.
```

Comment:
While we're at it, any function/method/filter/action documentation descriptions should start with a third-person verb. :)
```suggestion
 * Filters various image attributes to add the dominant color to the image.
```

Reply:
Thank you for the information.